### PR TITLE
[incubator/goldfish] Add Goldfish - Vault WebUI helm chart 

### PR DESCRIPTION
**Pull request by tuannvm**
## Why?
- Add [incubator/goldfish] - [goldfish](https://github.com/Caiyeon/goldfish) helm chart

### CODE REVIEW

**Comment by tuannvm** [comment]
Could you take a look if possible, @unguiculus ? Thanks!

**Comment by unguiculus** [comment]
We've started namespacing templates. Please apply this to your PR as well. See #1785.

File: incubator/goldfish/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
```
Comment (unguiculus) [inline incubator/goldfish/templates/_helpers.tpl]: W've started namespacing template. See #1785. Please apply this to your PR.

`{{- define "goldfish.name" -}}`
`{{- define "goldfish.fullname" -}}`

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/goldfish/templates/_helpers.tpl (hunk 1 above)
Comment (tuannvm) [reply incubator/goldfish/templates/_helpers.tpl]: thanks for the comment, I've fixed the issue mentioned above, please check again @unguiculus .

File: incubator/goldfish/values.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Default values for goldfish.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: quay.io/tuannvm/goldfish
```
Comment (unguiculus) [inline incubator/goldfish/values.yaml:6]: Make sure you provide a stable version in the future.

File: incubator/goldfish/README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+To install the chart with the release name `goldfish` in the default namespace:
+
+```bash
+helm install -n goldfish .
+```
+
+| Parameter                   | Description                        | Default                             |
+| --------------------------- | ---------------------------------- | ----------------------------------- |
+| `Name`                      | Name                               | `core`                              |
```
Comment (unguiculus) [inline incubator/goldfish/README.md]: Remove line

Comment (unguiculus) [reply incubator/goldfish/README.md]: Align table properly

**Review by unguiculus — CHANGES_REQUESTED** [review]
Just a few more things.

File: incubator/goldfish/values.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  tag: v0.7.3
+  pullPolicy: Always
```
Comment (unguiculus) [inline incubator/goldfish/values.yaml]: Default should be `IfNotPresent`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/goldfish/templates/secret.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ template "goldfish.fullname" . }}"
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "goldfish.fullname" . }}"
```
Comment (unguiculus) [inline incubator/goldfish/templates/secret.yaml]: app: "{{ template "goldfish.name" . }}"

**Review by unguiculus — CHANGES_REQUESTED** [review]
Discovered one more thing.

File: incubator/goldfish/templates/secret.yaml (hunk 5 above)
Comment (tuannvm) [reply incubator/goldfish/templates/secret.yaml]: thanks, good point.

File: incubator/goldfish/values.yaml (hunk 2 above)
Comment (tuannvm) [reply incubator/goldfish/values.yaml:6]: fixed

**Review by unguiculus — APPROVED** [review]

**Pull request merged**